src/firefly/js/visualize/region/RegionDescription.js

```diff
@@ -153,8 +154,7 @@ export function startRegionDes(regionType, cc, ptAry,
             sys = Point.IM_PT;
             des = `image;${DS9RegionName[regionType.key]}`;
             ptAry.forEach((pt) => {
-                var imgPt = cc.getImageCoords(pt);
-
+                const imgPt = cc.getFitsStandardImagePtFromInternal(cc.getImageCoords(pt)); // screen
```

Comment:
you can just say `cc.getFitsStandardImagePtFromInternal(pt)` and it should work.